events/main.py

```
@@ -0,0 +1,37 @@
+async def handler(websocket):
+    handlers = {
+        'meeting': MeetingHandler(websocket)
+    }
+
+    try:
+        async for message in websocket:
+            print(f"Received: {message}")
+            
+            event = json.loads(message)
+
+            handler = handlers.get(event.get('handler', None), None)
```

Ruff (0.15.18)

[warning] Use `handlers.get(event.get('handler', None))` instead of `handlers.get(event.get('handler', None), None)`

Replace `handlers.get(event.get('handler', None), None)` with `handlers.get(event.get('handler', None))`

(SIM910)